### DISCUSSION TITLE
无法Move index folder

### DISCUSSION THREAD

**Discussion by djyy1981** [xiaoyifang/goldendict-ng#1726] [Q&A]
![QQ图片20240813092913](https://github.com/user-attachments/assets/a703adf7-e8bc-4e91-b112-bc0e498d1267)
如图所示，无法Move index folder，按说明操作无论怎么也操作不成功，不知何故呢？

**Comment by xiaoyifang** [comment]
![image](https://github.com/user-attachments/assets/77ac9049-eabd-4ef6-b140-444856b1ce95)

**Reply by djyy1981** [reply]
谢谢回复，我是把原文件夹删除就成功了
